patches/api/0332-Update-Folder-Uses-Plugin-Name.patch

```diff
@@ -0,0 +1,86 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: samue <xemoryt@gmail.com>
+Date: Tue, 7 Sep 2021 18:11:37 +0100
+Subject: [PATCH] Update-Folder-Uses-Plugin-Name
+
+
+diff --git a/src/main/java/org/bukkit/plugin/SimplePluginManager.java b/src/main/java/org/bukkit/plugin/SimplePluginManager.java
+index 0e25119564dfa9cb12f3c5dc5f653d7f2c147a9d..26fa23f72a10d96701560554e7cbf6b89806a1ad 100644
+--- a/src/main/java/org/bukkit/plugin/SimplePluginManager.java
++++ b/src/main/java/org/bukkit/plugin/SimplePluginManager.java
+@@ -399,7 +399,7 @@ public final class SimplePluginManager implements PluginManager {
+     public synchronized Plugin loadPlugin(@NotNull File file) throws InvalidPluginException, UnknownDependencyException {
+         Validate.notNull(file, "File cannot be null");
+ 
+-        checkUpdate(file);
++        file = checkUpdate(file); //Paper - update the reference in case checkUpdate renamed it
```

Comment:
```suggestion
+        file = checkUpdate(file); // Paper - update the reference in case checkUpdate renamed it
```
Tiny nitpick here, add a space between `//` and `Paper` (same for the other two Paper comments)